contents/shells/bash.md

```diff
@@ -14,66 +14,60 @@
 
-<img src="images/shell-kernel.png" alt="shell-kernel" style="width:400px;"/>
+<img src="https://cdn-images-1.medium.com/max/1200/1*zEv6mAa2wzHnz4a5uWW4gw.png" alt="shell-kernel" style="width:300px;"/>
 
 Computer can be divided into 3 abstraction layers, hardware, kernel and applications. Users cannot control the hardware directly, instructions have to be given through kernel as kernel is the one that controls the hardware. 
-Shell is the interface through which we can pass instructions to kernel, and then it could be executed on the hardware. In this chapter, we will look into **Linux command-line based shell(bash)**.
+Shell is the interface through which we can pass instructions to kernel, and then it could be executed on the hardware. In this chapter, we will look into **Linux bash shell**.
 
-> Linux is the best-known and most-used open source operating system. As an operating system, Linux is software that sits underneath all of the other software on a computer, receiving requests from those programs and relaying these requests to the computer’s hardware.
+> Linux is the best-known and most-used open source operating system. As an operating system, Linux is software that sits underneath all of the other software on a computer, receiving requests from those programs and relaying these requests to the computer’s hardware. -- <cite>[What is Linux](https://opensource.com/resources/linux)</cite>
 
 [**bash**](https://www.gnu.org/software/bash/manual/html_node/) stands for **Bourne Again SHell**, an enhanced version of the original Unix Shell program. It has become the standard shell of various linux distributions. 
 
 ## Why use command-line based shell?
 
-A common question many people would ask is why would we learn command-line based shell as it seems to be very hard to use. There are tons of GUI tools out there, they seems to look more user-friendly and powerful. From my perspective, there are four major reasons for starting to use command line.
+A common question many people ask is why should we learn command-line based shell as it seems to be very hard to use. There are tons of Graphical User Interface (GUI) tools out there, that appear to be more user-friendly and powerful. From my perspective, there are four major reasons to use the command line:
 
-- **Remote Management**: Use over remote command-line shell makes it much easier to perform tasks remotely, such as when a developer is connecting from home. A remote command-line shell usually requires less bandwidth.
+- **Remote Management**: Use over remote command-line shell makes it much easier to perform tasks remotely, such as when a developer is connecting from home. A remote command-line shell usually requires less bandwidth, results in less lag such that we can do things faster.
```

Comment:
`Use over remote command-line` seems weird. Consider rewording?

Also: `bandwidth, results in less lag such that` -> `bandwidth, which results in less lag so that`